### ISSUE TITLE
Fern wifi cracker works if loginname is root.

### ISSUE THREAD

**Issue by joop28051952** [savio-code/fern-wifi-cracker#163]
That is my solution. i hope it do the same by you.

**Comment by kcee02** [comment]
what solution is that?

**Comment by joop28051952** [comment]
When I log in as a user, Fern-Wifi-Cracker does not find any stations. When I log in as root Fern works fine.

**Comment by kcee02** [comment]
Login as a root user? How ?
Like using the command: "sudo su"

**Comment by joop28051952** [comment]
No, when you login in Kali LinuxOutlook voor Android downloaden

**Comment by joop28051952** [comment]
Sorry for my langues, i am dutchOutlook voor Android downloaden
